src/guillo-gmi/components/context_toolbar.js

```diff
@@ -29,47 +24,24 @@ export function CreateButton(props) {
     })();
   }, [Ctx.path]);
 
-  useClickAway(ref, () => {
-    setState({ isActive: false });
-  });
-
-  const doAction = item => () => {
+  const doAction = item => {
     Ctx.doAction("addItem", { type: item });
     setState({ isActive: false });
   };
 
   // Implement some kind of filtering
-
-  const status = state.isActive
-    ? "dropdown is-right is-active"
-    : "dropdown is-right";
-
   return (
-    <div className={status} ref={ref}>
```

Comment:
@jordic I needed the same logic for this button:

![image](https://user-images.githubusercontent.com/13313058/73783902-c68bbd00-4794-11ea-8a91-ad1feea9af2f.png)

So I extracted outside in order to reuse it.